### DISCUSSION TITLE
time to transfer repository? from lavdotpet ➡ LibreQuake Org?

### DISCUSSION THREAD

**Discussion by lavenderdotpet** [MissLav/LibreQuake#230] [Ideas]
## been putting this off forever but should it be time to transfer the project to the librequake github org https://github.com/LibreQuake

![image](https://github.com/user-attachments/assets/0882e4f0-2e8a-4d2c-a943-3c204f067106)

## am a little worried about doing it, dont wanna mess up the repo

**Comment by MotoLegacy** [comment]
GitHub will redirect the old URLs to the organization and all of the git history will be preserved.

**Comment by pnahratow** [comment]
Why wasn't this mentioned sooner? I hinted at this in August

![image](https://github.com/user-attachments/assets/403027ba-0f52-4a1a-a660-b7c8ac1ccbdd)

**Comment by lavenderdotpet** [comment]
ive been meaning to do this for so long just was worried something would break so just kept putting it off

**Comment by MotoLegacy** [comment]
We should have clearer roadmaps for decisions like this in the future, this definitely should've been acted upon before the FlatPak package

**Comment by pnahratow** [comment]
I was missing the info that this was even the plan.
In any case I'm all for it. We will figure out the steps necessary to change the flatpak app id.

**Comment by lavenderdotpet** [comment]
> I was missing the info that this was even the plan. In any case I'm all for it. We will figure out the steps necessary to change the flatpak app id.

sorry 
i didnt realize that would mess up the flatpak

i could just not transfer it if its to much trouble

**Comment by MotoLegacy** [comment]
No one knew, I don't think. Hence our need for documentation

**Comment by lavenderdotpet** [comment]
> No one knew, I don't think. Hence our need for documentation

this is from the all the documentation is in my head days that we are still moving out of

**Comment by lavenderdotpet** [comment]
im just gonna close this 
disregard all of this

**Comment by MotoLegacy** [comment]
Re-opening. Dont be rash.

**Comment by lavenderdotpet** [comment]
no please stop reopening

**Comment by lavenderdotpet** [comment]
sorry about before logging off for now

**Comment by MotoLegacy** [comment]
Continuing on, I know there is also a website and a source file "attic". That would be migrated. Is there anything else?

**Comment by lavenderdotpet** [comment]
> Continuing on, I know there is also a website and a source file "attic". That would be migrated. Is there anything else?

website was already migrated a while back 

how do u feel about this being on the org repo list https://github.com/lavenderdotpet/pop.lmp_generator

**Comment by lavenderdotpet** [comment]
attic has now been migrated 
https://github.com/LibreQuake/lq-attic-and-sourcefiles

**Comment by pnahratow** [comment]
I want to reiterate that I am for this change. It looks better. I looks right. It should be like this. We will figure out the steps necessary to change the flatpak app id.

**Comment by Zungrysoft** [comment]
Do we know how many downloads we have from the flatpak? That could help inform this decision.

**Comment by lavenderdotpet** [comment]
> Do we know how many downloads we have from the flatpak? That could help inform this decision.

![image](https://github.com/user-attachments/assets/01ae9cde-3dc6-469f-8857-6a265b0c717b)

![image](https://github.com/user-attachments/assets/81cc7e68-f9c8-4f06-aea9-0f1e7ff8700c)


https://flathub.org/apps/io.github.lavenderdotpet.LibreQuake
edit: alot more then i was expecting

**Comment by MotoLegacy** [comment]
@pnahratow already volunteered to do whatever maintenance required and signed off on this decision. I don't think there's anything else concerning the Flatpak that could be considered a blocker
